en/components/animations.md

```diff
@@ -78,10 +78,53 @@ useAnimation(fadeIn, {
     }
 });
 ```
+## SCSS Animations
+
+### Keyframes Mixin
+
+The Ignite UI for Angular [keyframes]({environment:sassApiUrl}/index.html#mixin-keyframes) mixin is used to register new keyframes animations. The mixin takes the name of a keyframes animation as a parameter and adds it to the global keyframe register list. Keyframes selectors for the animation steps along with CSS styles for the keyframes are defined inside the body of the mixin.  
```

Comment:
Please say why adding the keyframes to the global register is useful. In that way, the keyframes will not be duplicated in the exported CSS when including the same keyframes animation several times. For instance, doing this:
```scss
  @include fade-in();
  @include fade-in();
```
Will result in only one `@keyframes fade-in { ... }` added to the produced CSS.